nebula/ui/components/VPNGuideList.qml

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+
+import Mozilla.VPN 1.0
+import Mozilla.VPN.qmlcomponents 1.0
+import components 0.1
+
+ColumnLayout {
+    property string title
+    property string description
+
+    property var customGuideFilter: () => true
+    property var count: guideRepeater.count
+
+    Layout.alignment: Qt.AlignTop
+    Layout.fillWidth: true
```

Comment:
I would remove these for the same reason as [this](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4221/files/4b464c41374fb57dd2b80483790cffaf01a29454#r955652129)